tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/SlackNotificationPayloadContent.java

```diff
@@ -77,7 +73,7 @@
     private ArrayList<String> failedTestNames = new ArrayList<String>();
 
     public SlackNotificationPayloadContent(){
-
+		payloadCommits = new PayloadContentCommits();
```

Comment:
you could move this declaration to the variable initializer at the top of the file and save the additional 3 declarations.

You could also use this() to call the next constructor, which would also be a better practice. (I know this is not the case right now, but since the title is called refactoring it might make sense ;) )